### DISCUSSION TITLE
HomeAssistant - zwave-js-ui - apt update info

### DISCUSSION THREAD

**Discussion by bjh45964** [zwave-js/zwave-js-ui#2772] [Ideas]
First off, THANK YOU!  I'm using zwave-js-ui with 2 x raspberry pi's with zooz 700 S2 sticks to monitor zwave devices between two buildings.  Integration with HomeAssistant works amazing, and zwave-js-ui is amazing.

Have you ever considered including some data from a host OS in the zwave-js-ui interface?  I'm trying to find a way to monitor for package updates on the Pi's, but I'd like those updates to be exposed to HA through the zwave-js-ui integration.  My thought process is that the package is so seamless right now, zwave setup is very much appliance like, so I'd like the zwave-js-ui integration to be "aware" that there are OS updates.  Generally speaking, with many major HA releases, my zwave-js-ui install breaks because HA needs a current version of zwave-js-ui.

I realize this is probably counter to the idea of keeping the package abstract from specific hardware, but I would imagine rPi is a very common setup for zwave-js-ui.

My idea would be for zwave-js-ui to report either the number of updates available from the host OS, or to report on the availability of updates for itself via apt.  Maybe could be tied into MQTT somehow, to allow host to send metadata to zwave-js-ui via MQTT.

**Comment by robertsLando** [comment]
> My idea would be for zwave-js-ui to report either the number of updates available from the host OS, or to report on the availability of updates for itself via apt.

Zwave-js UI could be installed almost everywhere so this is a too specific use case, I would suggest to use dedicated tools for that like rpi [webmin](https://pimylifeup.com/raspberry-pi-webmin/)
